src/main/java/io/antmedia/AntMediaApplicationAdapter.java

```diff
@@ -895,6 +895,7 @@ public void closeStreamFetchers() {
 			Queue<StreamFetcher> fetchers = streamFetcherManager.getStreamFetcherList();
 			for (StreamFetcher streamFetcher : fetchers) {
 				streamFetcher.stopStream();
+				fetchers.remove(streamFetcher);
```

Comment:
Are you it does not throw concurrent modification exception?
Please add more than one fetcher in your tests and check if it throws concurrentmodifiction exception?

Reply:
> Are you it does not throw concurrent modification exception?

Yes, I tested a lot of different scenarios. 

> Please add more than one fetcher in your tests and check if it throws concurrent modification exception?

I added a try-catch exception in tests and closeStreamFetchers function.